spec/models/user_spec.rb

```diff
@@ -412,7 +412,7 @@
     it 'returns all associated affiliate display names' do
       affiliates(:power_affiliate).users << user
       affiliates(:basic_affiliate).users << user
-      expect(user.affiliate_names.split(',').sort).to eq(%w[noaa.gov nps.gov])
+      expect(user.affiliate_names.split(', ').sort).to eq(%w[noaa.gov nps.gov])
```

Comment:
OK as-is, but we could also simplify this spec, which obscures the fact that the method is intended to return a formatted string:
```ruby
expect(user.affiliate_names).to eq ('noaa.gov, nps.gov')
```